impl/src/main/kotlin/nl/nuts/consent/bridge/corda/CordaManagedonnectionFactory.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Nuts consent bridge
+ * Copyright (C) 2019 Nuts community
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package nl.nuts.consent.bridge.corda
+
+import nl.nuts.consent.bridge.ConsentBridgeRPCProperties
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.config.AbstractFactoryBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * abstraction for creating Corda RPC connection,
+ */
+class CordaManagedConnectionFactory : AbstractFactoryBean<CordaManagedConnection>() {
```

Comment:
Would this be a candidate for a health indicator so we can monitor Corda connections?

Reply:
There is a dedicated one in CordaFlowHealthIndicators: CordaConnectionHealthIndicator